client/src/common/hooks/useFullscreen.js

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useEffect, useState } from 'react';
+
+
+export default function useFullscreen() {
+  const [isFullScreen, setFullScreen] = useState(document?.fullscreenElement);
+
+  useEffect(() => {
+    const handleChange = () => {
+      setFullScreen(document?.fullscreenElement);
+    };
+    document.addEventListener('fullscreenchange', handleChange, false);
+    document.addEventListener('resize', handleChange, false);
```

Comment:
Do we need the third argument here? False is the default value. Maybe you wanted to use `{ passive: true }` ?